DavidGonzalezZumaya/Collections/src/RansomNote/Result.java

```diff
@@ -0,0 +1,31 @@
+package RansomNote;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Result {
+
+    public static void main(String[] args){
+
+        Scanner in = new Scanner(System.in);
+        int magazineSize = in.nextInt();
+        int noteSize = in.nextInt();
+
+        List<String> magazine = new ArrayList<>();
+
+        for (int magazineIndex = 0; magazineIndex < magazineSize; magazineIndex++) {
+            String word = in.next();
+            magazine.add(word);
+        }
+
+        List<String> note = new ArrayList<>();
```

Comment:
It is recommended that the name of the collections and maps start with singular name and them the name of the interface, example: numberList